docs/Getting Started.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started
+
+## Clone Serai
+```
+git clone https://github.com/serai-dex/serai.git
+```
+
+## Build and Run Serai

```

Comment:
This isn't a helpful header when it then includes:

- Rust
- Solidity
- Random system packages
- Build
- Run
- Help

Which is the entire document.